src/main/java/com/github/instagram4j/instagram4j/requests/fbsearch/places/SearchLocationRequest.java

```diff
@@ -0,0 +1,60 @@
+package com.github.instagram4j.instagram4j.requests.fbsearch.places;
+
+import com.github.instagram4j.instagram4j.IGClient;
+import com.github.instagram4j.instagram4j.requests.IGGetRequest;
+import com.github.instagram4j.instagram4j.responses.fbsearch.places.SearchLocationResponse;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+import java.util.Optional;
+
+@AllArgsConstructor
+@NoArgsConstructor
+public class SearchLocationRequest extends IGGetRequest<SearchLocationResponse> {
```

Comment:
This request should be placed in "fbsearch" package (Not fbsearch.places)

Additionally, the request must also be named FbsearchPlacesRequest (It needs to follow the naming of the path and then Request at the end).

Reply:
fixed